src/js/front-end/setup/general/replay-block.js

```diff
@@ -0,0 +1,11 @@
+import { systemState } from "../../front-end.js"
+
+const allowedDuringReplay = {
+    "action": ["changeCardBack", "viewDeck", "lookAtCards", "stopLookingAtCards", "revealCards", "hideCards", "revealShortcut", "hideShortcut", "lookShortcut", "stopLookingShortcut"],
+    "keybind": ['v','c','z','r','f','esc'],
+    "contextMenu": ['lookPrizesButton','revealHidePrizesButton','lookHandButton','prizesHeader','handHeader','deckHeader','boardHeader'],
+};
+
+export const replayBlock = (type, value, isFromReplay = false) => {
```

Comment:
i think i'd rename this to `isActionBlockedByReplay` for clarity.

Reply:
or even just `isBlockedByReplay`

Reply:
Done in [6621d23](https://github.com/xxmichaellong/ptcg-sim/pull/19/commits/6621d237b246d021b4310c791c6d1249b9379059)